.storybook/main.js

```diff
@@ -0,0 +1,12 @@
+/* jshint node: true */
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
Userei solo typescript
```suggestion
    "../src/**/*.stories.mdx",
    "../src/**/*.stories.@(ts|tsx)"
```